django_redshift_backend/base.py

```diff
@@ -159,8 +159,10 @@ def create_model(self, model):
                 definition += " %s" % col_type_suffix
             params.extend(extra_params)
             # FK
-            if field.rel and field.db_constraint:
-                to_table = field.rel.to._meta.db_table
+            if field.remote_field and field.db_constraint:
+                to_table = field.remote_field.to._meta.db_table
+                to_column = field.remote_field.to._meta.get_field(
+                    field.remote_field.field_name).column
                 to_column = field.rel.to._meta.get_field(field.rel.field_name).column
```

Comment:
I think this line must be removed because new code is provided at one line before.

Reply:
oh... sorry